FirebaseAuth/Tests/Unit/AuthTests.swift

```diff
@@ -16,8 +16,67 @@ import Foundation
 import XCTest
 
 @testable import FirebaseAuth
+// TODO(ncooke3): Remove below import after `FIRAuth.[hm]` has been ported.
+// Note– This will break building with SPM. There is a way to achieve the same
+// thing with SPM mixed targets–– I'm just going to port `FIRAuth` next.
+import FirebaseAuth_Private
+
 import FirebaseCore
 
+/// A fake storage instance that imitates the system keychain while storing data in-memory.
+final class FakeAuthKeychainServices: NSObject, AuthStorage {
```

Comment:
Can FakeAuthKeychainServices be in its own file?

Reply:
yep. will do.

Reply:
Added FirebaseAuth/Tests/Unit/**Fakes/FakeAuthKeychainServices**.swift in 9093221